src/Core/src/RuntimeFeature.cs

```diff
@@ -55,6 +56,11 @@ internal static bool IsShellSearchResultsRendererDisplayMemberNameSupported
 			AppContext.TryGetSwitch("Microsoft.Maui.RuntimeFeature.IsImplicitCastOperatorsUsageViaReflectionSupported", out bool isSupported)
 				? isSupported
 				: IsImplicitCastOperatorsUsageViaReflectionSupportedByDefault;
+
+		internal static bool AreBindingInterceptorsEnabled =>
+			AppContext.TryGetSwitch("Microsoft.Maui.RuntimeFeature.AreBindingInterceptorsEnabled", out bool areEnabled)
```

Comment:
Should the switch name end in `Supported` such as `Microsoft.Maui.RuntimeFeature.IsImplicitCastOperatorsUsageViaReflectionSupported`?

Reply:
Sounds good

Reply:
Although now that I'm thinking about it more, maybe `Enabled` makes more sense since the feature can be enabled or disabled, but it's supported the whole time. In other words, it's not a limitation of the platform when you disable it.

Reply:
Scratch what I said, there's an unwritten convention to base the naming on the default value: in this case, the default value is `true` and you need to turn it off so it should be called `Microsoft.Maui.RuntimeFeature .AreBindingInterceptorsSupported` + `_MauiBindingInterceptorsSupport`. I will update the PR now.